enhancements/two-nodes-openshift/2no_template.md

```diff
@@ -0,0 +1,589 @@
+- Does this enhancement modify the behaviour of existing resources, especially those owned
+  by other parties than the authoring team (including upstream resources), and, if yes, how?
+  Please add those other parties as reviewers to the enhancement.
+
+  Examples:
+  - Adds a finalizer to namespaces. Namespace cannot be deleted without our controller running.
+  - Restricts the label format for objects to X.
+  - Defaults field Y on object kind Z.
+
+Fill in the operational impact of these API Extensions in the "Operational Aspects
+of API Extensions" section.
+
+### Topology Considerations
+
+2NO represents a new topology, and is not appropriate for use with HyperShift, SNO, or MicroShift
+
+#### Hypershift / Hosted Control Planes
+
+Are there any unique considerations for making this change work with
+Hypershift?
+
+See https://github.com/openshift/enhancements/blob/e044f84e9b2bafa600e6c24e35d226463c2308a5/enhancements/multi-arch/heterogeneous-architecture-clusters.md?plain=1#L282
+
+How does it affect any of the components running in the
+management cluster? How does it affect any components running split
+between the management cluster and guest cluster?
+
+#### Standalone Clusters
+
+Is the change relevant for standalone clusters?
+
+#### Single-node Deployments or MicroShift
+
+How does this proposal affect the resource consumption of a
+single-node OpenShift deployment (SNO), CPU and memory?
+
+How does this proposal affect MicroShift? For example, if the proposal
+adds configuration options through API resources, should any of those
+behaviors also be exposed to MicroShift admins through the
+configuration file for MicroShift?
+
+### Implementation Details/Notes/Constraints
+
+#### Installation flow
+1. We’ll set up Pacemaker and Corosync on RHCOS using MCO layering.
+   * [TBD extend more]
+2. Install an “SNO like” first node using a second bootstrapped node.
+   * This is somewhat similar what is done in SNO CI in AWS (up until the part the bootstrapped node is removed) and is possible because CEO can distinguish the bootstrapped node as a special use case, thus enabling its removal without breaking the etcd quorum for the remaining node.
+   We should be safe after [MGMT-13586](https://issues.redhat.com/browse/MGMT-13586) which makes the installer wait for the bootstrap etcd member to be removed first before shutting it down.
+3. After the bootstrapped node is removed add it to the cluster as a “regular” node.
+4. Switch CEO to “2NO” mode (where it does not manage etcd) and remove the etcd static pods
+   * [TBD localized/global switch]
+   * This is done because we want to allow simpler maintenance and keeping  some of CEO functionality (defragmentation, cert rotation ect…)
+5. Configure Pacemaker/Corosync to manage etcd/kubelet/cri-o
+6. [TBD storage]
+
+#### Fencing / quorum management
+Fencing quorum is managed by corosync and etcd  will be managed by Pacemaker which will force the etcd quorum.
```

Comment:
I'm confused by this sentence. Do you mean "fencing quorum" or "fencing and quorum".
The former would be a new term that's not been defined, and the second would be incorrect (corosync does not manage fencing).

Suggest breaking this into two independent sentences

Reply:
```suggestion
Corosync will manage the quorum.
Pacemaker will apply node fencing and etcd management (promote/demote) according to Corosyncs quorum
```
I've tried to rephrase, let me know if that works

Reply:
There's two types of quorum.

Maybe: Corosync will provide an independent definition of quorum

Reply:
Resolved